src/domains/ecommerce-common/00_alerts.tf

```diff
@@ -254,4 +254,46 @@ AzureDiagnostics
     operator  = "GreaterThanOrEqual"
     threshold = 2
   }
+}
+
+# eCommerce user stats service availability
+resource "azurerm_monitor_scheduled_query_rules_alert" "ecommerce_user_stats_service_availability_alert" {
+  count = var.env_short == "p" ? 1 : 0
+
+  name                = "ecommerce-user-stats-service-availability-alert"
+  resource_group_name = azurerm_resource_group.rg_ecommerce_alerts[0].name
+  location            = var.location
+
+  action {
+    action_group           = [data.azurerm_monitor_action_group.email.id, data.azurerm_monitor_action_group.slack.id]
+    email_subject          = "[eCommerce] User stats service availability less that 99%"
```

Comment:
specify that it is not the availability of the `user-stats-service` but only of the PUT api

Reply:
If we add more APIs to this group in the future, will we edit this description again?
Or we could modify the description to explain that they refer only to api reachable between microservices.

Reply:
At the moment `where ` condition detect all user-stats-service api (only PUT is exposed.

`where url_s startswith 'https://api.platform.pagopa.it/ecommerce/user-stats-service/v1'`

Reply:
but in this API group there is no `GET user-stats`, which is part of the service. If all GET fail but PUT continue to work, this alert does not fire but we can say that `[eCommerce] User stats service availability less than 99%` is ok?

Reply:
anyway it's not blocking for me

Reply:
done here 1ea2363a442065057f721759d9ad082b8adb20c7